http/vibe/http/client.d

```diff
@@ -1118,7 +1118,7 @@ final class HTTPClientResponse : HTTPResponse {
 }
 
 /** Returns clean host string. In case of unix socket it performs urlDecode on host. */
-private auto getFilteredHost(URL url)
+auto getFilteredHost(URL url)
```

Comment:
I'd leave this as `package` for now, because the place here isn't really fitting for a public API.